IgOmeProfiling_pipeline.py

```diff
@@ -42,8 +49,9 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
 
         module_parameters = [fastq_path, first_phase_output_path, first_phase_logs_path,
                              barcode2samplename_path, left_construct, right_construct,
-                             max_mismatches_allowed, min_sequencing_quality, first_phase_done_path,
-                             '--gz' if gz else '', f'--error_path {error_path}', '-v' if verbose else '']
+                             max_mismatches_allowed, min_sequencing_quality, first_phase_done_path, 
+                             '--check_files' if check_files else '', '--gz' if gz else '',
```

Comment:
If we already know the files are valid why are we asking other parts of the pipeline to recheck it? The name is confusing.

Each module wrapper should have a flag whether to validate files.
The default can be true and if validated the files in main pipeline then value should be false